property-scraper.js

```diff
@@ -192,17 +194,33 @@ class PropertyScraper {
         });
     };
 
-    static parseByTMK(tmks){
+    static scrapeByTMKs(tmks){
         const base = "mongodb://127.0.0.1:27017/test1",
             collectionName = "hnl_county_data";
         for (let i = 0; i < tmks.length; i++) {
+            if (i % 1000 === 0) {
+                console.log(`iteration ${i}`);
+            }
             PropertyScraper.getAllData(tmks[i], function(object) {
                 PropertyScraper.insertOneInDB(object, base, collectionName)
             });
-            // PropertyScraper.insertOneInDB(object, base, collectionName);
         }
     }
 
+    static scrapeWithCSV(filename) {
+        let tmks = [];
+
+        fs.createReadStream(filename)
+            .pipe(csv())
+            .on('data', function (data) {
```

Comment:
since you only use one piece of data, TMK. You can use destructuring assignment: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
```
.on('data', function ({ TMK }) {
  tmks.push(+`${TMK}0000`);
})
```

Reply:
Why need + in the push argument?